index.html

```diff
@@ -6883,6 +7140,303 @@ <h2 id="MediaStreamAudioSourceNode">
+          Remove the oldest element of <em>Q<sub>rendering</sub></em>.
+          <li>Let <em>Q<sub>rendering</sub></em> be an empty <a>control message
+          queue</a>. <a>Atomically</a> <a>swap</a> <em>Q<sub>rendering</sub></em> and
+          <em>Q</em>
+          </li>
+          <li>Until there is no more messages in <em>Q<sub>rendering</sub></em>,
+          execute the following steps:
+            <ol>
+              <li>Execute the asynchronous section of the <a>oldest message</a>
+              of <em>Q<sub>rendering</sub></em>.
+              </li>
+              <li>Remove the <a>oldest message</a> of
+              <em>Q<sub>rendering</sub></em>.  </li>
+            </ol>
+          </li>
+          <li>Order the <a>AudioNode</a>s of the <a>AudioContext</a> to be
+          processed.
+            <ol>
+              <li>Let <em>ordered</em> be an empty list of <a>AudioNode</a>s.
+              It will contain an ordered list of <a>AudioNode</a>s when this
+              ordering algorithm terminates.
+              </li>
+              <li>Let <em>nodes</em> be the set of all nodes belonging to an
+              <a>AudioContext</a>.
+              </li>
+              <li>Let <em>cycle breakers</em> be an empty set of
+              <a>DelayNode</a>s. It will contain all the <a>DelayNode</a>s that
+              are part of a cycle.
+              </li>
+              <li>For each <a>AudioNode</a> <em>node</em> of <em>nodes</em>
+                <ol>
+                  <li>
+                    <em>node</em> is a <a>DelayNode</a> part of a cycle, add it
+                    to <em>cycle breakers</em> and remove it from
+                    <em>nodes</em>.
+                  </li>
+                </ol>
+              </li>
+              <li>If <em>nodes</em> contains cycles, mute all the
+              <a>AudioNode</a>s that are part of this cycle, and remove them
+              from <em>nodes</em>.
+              </li>
+              <li>While there are unmarked <a>AudioNode</a>s in <em>nodes</em>:
+                <ol>
+                  <li>Choose an <a>AudioNode</a> <em>node</em> in
+                  <em>nodes</em>
+                  </li>
+                  <li>
+                    <a>Visit</a> <em>node</em>.
+                  </li>
+                </ol><dfn data-lt="Visit">Visiting a node</dfn> <em>node</em>
+                mean performing the following steps:
+                <ol>
+                  <li>If <em>node</em> is marked, abort these steps.
+                  </li>
+                  <li>For each <a>AudioNode</a> <em>input</em> connect to <em>
+                    node</em>:
+                    <ol>
+                      <li>
+                        <a>Visit</a> <em>input</em>.
+                      </li>
+                    </ol>
+                  </li>
+                  <li>Mark <em>node</em>.
+                  </li>
+                  <li>Add <em>node</em> to the beginning of <em>ordered</em>.
```

Comment:
This algorithm seems to result in nodes being ordered with the _destination_ end of the graph at the _beginning_ of the ordered list. That seems the reverse of what is intended, give the subsequent section that appears to be intended to work through the graph from sources towards destinations, making buffers available for reading by downstream nodes.